src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,68 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
-        Movie fastAndFurious = new Movie("Fast and Furious");
+    public static void main(String[] args) {
+        Movie fastAndFurious = new Movie();
+        fastAndFurious.setTitle("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
+        Movie homeAlone = new Movie();
+        homeAlone.setTitle("Home alone");
+        homeAlone.setDescription("Christmas comedy");
+
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
         movieService.add(fastAndFurious);
+        movieService.add(homeAlone);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        CinemaHall firstHall = new CinemaHall();
+        firstHall.setCapacity(100);
+        firstHall.setDescription("First hall");
+        CinemaHall secondHall = new CinemaHall();
+        secondHall.setCapacity(200);
+        secondHall.setDescription("Second hall");
+
+        CinemaHallService cinemaHallService =
+                (CinemaHallService) injector.getInstance(CinemaHallService.class);
```

Comment:
move all injector.getInstance to the top of main method

Reply:
for what?)